### DISCUSSION TITLE
deep-learning/convolutional/resnet

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#87] [Announcements]
# deep-learning/convolutional/resnet

ResNet解决了深度卷积网络难以训练的问题。残差学习的原理。ResNet中Bottleneck和Basic有什么区别？

https://lulaoshi.info/deep-learning/convolutional/resnet.html

**Comment by mebest100** [comment]
恒等映射=》"什么都不学"，的确就是残差网络的精髓

**Comment by Nilebaba** [comment]
很有用，感谢
